app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = [Person(name=person["name"], age=person["age"])
+                   for person in people]
```

Comment:
The list comprehension correctly creates `Person` instances, but ensure that the input list `people` contains dictionaries with the keys `name` and `age` to avoid KeyError.